src/SonarQube.Client/Models/Projects.cs

```diff
@@ -19,17 +19,37 @@
  */
 
 using System;
-using System.Diagnostics;
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service
+namespace SonarQube.Client.Models
 {
-    [DebuggerDisplay("Name: {Name}, Key: {Key}")]
-    internal class ProjectInformation
+    public class SonarQubeProject
     {
         // Ordinal comparer should be good enough: http://docs.sonarqube.org/display/SONAR/Project+Administration#ProjectAdministration-AddingaProject
         public static readonly StringComparer KeyComparer = StringComparer.Ordinal;
 
+        public string Key { get; }
+        public string Name { get; }
+
+        public SonarQubeProject(string key, string name)
+        {
+            Key = key;
+            Name = name;
+        }
+
+        public static SonarQubeProject FromDto(ProjectDTO dto)
+        {
+            return new SonarQubeProject(dto.Key, dto.Name);
+        }
+
+        public static SonarQubeProject FromDto(ComponentDTO dto)
+        {
+            return new SonarQubeProject(dto.Key, dto.Name);
```

Comment:
_FromDto(ComponentDTO)_ and _FromDto(ProjectDTO_ set the same key on _SonarQubeProject_.
I thought the SQ project key was different from the SQ component key (and that the component key was a composite key that started with the project key)?

I guess the important thing is - do we need to be able to distinguish between project and component keys on the client?